### ISSUE TITLE
No Access to engine on iPad-2 Wifi-Only

### ISSUE THREAD

**Issue by mfoster98** [Scirra/Construct-3-bugs#363]
## Problem description

*Whenever I load the editor.construct.net it shows the beginning logo animation but then doesn't show any loading bar or loading progress.  I am using the chrome app for iOS 9.3.5 and Safari iOS 9.3.5*

## Steps to reproduce

1.Launch the engine on iPad-2 Wifi Only iOS 9.3.5
2.Witness the failure to load

## Observed result

*No action after the beginning logo*

## Expected result

*An awesome game engine to appear!*

## Affected browsers

* Chrome: yes
* Safari: yes         :Both Zoom Levels are 100%

## System details

*If you see a crash report dialog, please copy and paste it to where it says "PASTE HERE" below. Otherwise please go to **Menu** -> **About** -> **Platform information** and paste that information there instead.*

<details><summary>View details</summary>

I can't access my platform information to paste or show any crash report.  However I can provide you with a link to my iPads specifications [here](http://www.everymac.com/systems/apple/ipad/specs/apple-ipad-2-wi-fi-specs.html).
</details>

**Comment by shortercode** [comment]
Construct 3 doesn't work on Chrome for iOS. It's basically just a re-skinned safari, and safari is running a bit behind the other browsers for several important features.

There should be a warning message, can you not see it on your device?

**Comment by mfoster98** [comment]
I dont see any error messages just the logo

**Comment by shortercode** [comment]
Ah okay I've tried a device on 9.x.x and can see what you mean. It's not magically going to gain support for C3 but we're going to fix the error dialog.

Sorry to say this but given the lack of basic ES6 features in iOS 9.x.x ( which we embraced heavily in development of C3 )  I don't think any device running it will ever be able to support C3. We're still hopeful with newer version of iOS that we'll get it working, but we're waiting on apple for a few things still...
